search-app/src/cmr/search/api/concepts_search.clj

```diff
@@ -315,11 +309,10 @@
     (find-tiles ctx params)))
 
 (def data-json-routes
-  "config-enabled route for data.json. Socrata does not support harvesting from
+  "Route for data.json response. Socrata does not support harvesting from
```

Comment:
Now that this route is official, I think we should add it in the api doc or at least somewhere in README so that it does not get lost as time goes on.

Reply:
hmm do you think the search-app readme would be okay or should i put it in the top level cmr readme?

Reply:
I am actually leaning towards in the search api doc. We document the facet v2 api in api doc even though it is not supposed to be called manually. This api is similar to that. search-app README is fine too if we don't want to expose this route in the public api doc. Definitely not in the top level CMR README.

Reply:
I was more leaning towards the readme because I have no idea why anyone other than us would actually want to use it.

Reply:
Yeah, I see your point. I am fine with the search-app README.